docs/changelog.rst

```diff
@@ -2,6 +2,22 @@
 Changelog
 =========
 
+* Add :attr:`time_machine.naive_mode` to control how time-machine interprets naive datetimes.
+
+  The default mode is ``MIXED``, which preserves existing behaviour: naive ``datetime`` objects and ``date`` objects are interpreted as UTC, while naive datetime strings are interpreted as local time.
+
+  Three alternative modes are available:
+
+  * ``UTC``: naive datetimes are always interpreted as UTC.
+  * ``LOCAL``: naive datetimes are interpreted as local time, matching Python's default semantics, and freezegun.
+  * ``ERROR``: naive datetimes raise a ``RuntimeError``, ensuring your tests are isolated from the current timezone.
+
+  It’s recommended you use ``LOCAL`` or ``ERROR`` to avoid confusion around naive datetimes.
```

Comment:
```suggestion
.. note::

   It’s recommended you use `LOCAL` or `ERROR` to avoid confusion around naive datetimes.
```
Consider using `.. note::` for emphasizing recommendations, as it aligns with the project's documentation style.

Reply:
Yeah good shout